plover_casecat_dictionary.py

```diff
@@ -1,45 +1,45 @@
-
-from plover.steno_dictionary import StenoDictionary
-from plover.steno import normalize_steno
-from struct import unpack
 import re
+from struct import unpack
+from plover.steno_dictionary import StenoDictionary
 
 KEYS = 'STKPWHRAO*EUFRPBLGTSDZ'
 
 # I admit this is a wild guess
 ENCODING = 'latin-1'
 
-def _tidy_translation(t):
 
-    result = t.decode(ENCODING)
+class CaseCatDictionary(StenoDictionary):
 
-    curly_brackets = False
+    @staticmethod
+    def _tidy_translation(translation):
 
-    # these codes have been figured out
-    # by inspecting saved dictionaries;
-    # they may be wrong.
+        result = translation.decode(ENCODING)
 
-    if result.startswith('\x10\x81'):
-        # prefix
-        result = '^' + result
-        curly_brackets = True
+        curly_brackets = False
 
-    if result.endswith('\x10\x01'):
-        # suffix
-        result = result + '^'
-        curly_brackets = True
+        # these codes have been figured out
+        # by inspecting saved dictionaries;
+        # they may be wrong.
 
-    if '\x0E' in result:
-        # cap up
-        result += '{-|}'
+        if result.startswith('\x10\x81'):
+            # prefix
+            result = f'^{result}'
+            curly_brackets = True
 
-    if curly_brackets:
-        result = '{'+result+'}'
+        if result.endswith('\x10\x01'):
+            # suffix
+            result = f'{result}^'
+            curly_brackets = True
 
-    result = re.sub(r'[^ -~]', '', result)
-    return result
+        if curly_brackets:
+            result = '{' + result + '}'
 
-class CaseCatDictionary(StenoDictionary):
+        if '\x0E' in result:
+            # cap up
+            result += '{-|}'
+
+        result = re.sub(r'[^ -~]', '', result)
```

Comment:
What's this bit come from?

Reply:
I really can't remember at this distance :/ It's probably unnecessary though.